tasks/main.yml

```diff
@@ -65,6 +65,10 @@
         rsyslog_purge_original_conf: yes
       when: logging_purge_confs | d(false)
 
+    - name: Set the value of logging_system_log_dir to __rsyslog_system_log_dir
+      set_fact:
+        __rsyslog_system_log_dir: "{{ logging_system_log_dir }}"
```

Comment:
Future work for another PR - this and many of these could be rsyslog role parameters, instead of using `set_fact`.

Reply:
@richm, is this diff close to what you suggested?
https://github.com/nhosoi/logging/compare/rsyslog_system_log_dir...nhosoi:remove_set_fact

If it looks ok, may I add the commit to this pr?  Or had I better open another pr just for that once this pr is merged?  Thanks.

Reply:
Looks ok to me - it seems a bit odd to have "private" parameters passed to the role as vars, but these are internal interfaces not meant for end users anyway.

Reply:
Thank you, @richm.  Let me add the patch to this pr, then. :)